test/package.json

```diff
@@ -7,18 +7,42 @@
     "coverage-ci": "node --preserve-symlinks test.js test --framework karma"
   },
   "dependencies": {
-    "axios": "^0.19.2",
-    "axios-mock-adapter": "^1.15.0",
-    "bedrock": "^3.0.0",
-    "bedrock-karma": "^2.1.0",
-    "bedrock-test": "^5.0.0",
+    "@digitalbazaar/ed25519-verification-key-2018": "^3.0.0",
+    "@digitalbazaar/http-client": "^1.0.0",
+    "base58-universal": "^1.0.0",
+    "base64url-universal": "^1.1.0",
+    "bedrock": "^4.1.1",
+    "bedrock-karma": "^2.2.0",
+    "bedrock-test": "^5.3.2",
     "bedrock-web-data-hub": "^1.0.1",
     "bedrock-web-kms": "^2.0.0",
-    "bedrock-web-mock-data-hub-storage": "^2.0.0",
     "bedrock-web-mock-kms-http": "^2.0.1",
-    "bedrock-web-store": "^1.1.1",
+    "bedrock-web-store": "^1.2.0",
     "bedrock-web-vc-store": "file:..",
-    "edv-client": "^4.0.1",
-    "pretender": "^3.4.3"
+    "crypto-ld": "^3.9.0",
+    "did-context": "^2.0.0",
+    "did-method-key": "^0.7.0",
+    "edv-client": "^8.0.0",
+    "isomorphic-webcrypto": "^2.3.8",
+    "mocha": "^8.3.2",
+    "node-forge": "^0.10.0",
+    "path-to-regexp": "^3.2.0",
+    "route-params": "^0.2.0",
+    "sinon": "^9.2.4",
+    "tweetnacl": "^1.0.3",
+    "uuid-random": "^1.3.2",
+    "webkms-client": "^4.0.0"
+  },
+  "devDependencies": {
+    "webpack-cli": "^4.6.0"
+  },
```

Comment:
`devDependencies` in a test folder are not really a thing.  Please just make this a regular dep.

Reply:
Removed here 10ac153.